generators/app/templates/src/containers/Profile/children/auto-save.component.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect, memo } from 'react';
```

Comment:
Does this file differ from the existing auto-save component? If it's just copy/paste, can we reuse the other one instead of a new component?

Reply:
Yes James, I did move the autosave spinner to a new global component and call it in both of the pages.